### PR TITLE
Feature/inventory bar widget

### LINKED ISSUE #15: Feature/Game-board
Ce widget représente le "game manager". C'est lui qui va gérer l'affichage et les interactions de la partie. Il prend toute la largeur de l'écran et place les différents éléments.

image

### Il possède :
- [x] Une carte avec son terrain
- [x] Une collection de pions joueur avec leurs positions
- [x] Une barre d'inventaire
- [ ] 1 barre de statistique pour les points de vie
- [ ] 1 barre de statistique pour les points d'actions
- [ ] Un élément indiquant à qui est le tour de jouer
- [x] Un menu présentant les statistiques des joueurs adverses
- [ ] Un bouton pour terminer son tour manuellement

### LINKED ISSUE #16: Feature/inventory-bar-widget
Ce widget représente la barre d'inventaire.

image

### Il possède :
- [x] Une collection d'objet

### Il relève les évènements :
- [x] `@Click` pour signaler la sélection d'un objet

### PR DESCRIPTION
**Pull request by Laroche-Nicolas**

### CODE REVIEW

**Review by zaktolba — APPROVED** [review]

File: src/reusables/InventoryBarWidget.vue [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  data() {
+    return {
+      inventory: [{id:"1234", title: "Objet 1", imageURL: "https://via.placeholder.com/80"},
+        {id:"12323", title: "Objet 2", imageURL: "https://via.placeholder.com/80"},
+        {id:"123456cc2", title: "Objet 3", imageURL: "https://via.placeholder.com/80"}],
+    }
+  },
+  methods: {
+    clickOnItem(item) {
+      console.log(`l'id de l'objet est ${item.id}`)
```
Comment (MaxenceMottard) [inline src/reusables/InventoryBarWidget.vue]: Could you emit an event with this.$emit() function.
Documentation: https://fr.vuejs.org/v2/guide/components-custom-events.html#Noms-d’evenements

**Review by MaxenceMottard — CHANGES_REQUESTED** [review]

File: src/reusables/InventoryBarWidget.vue [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  }
+}
+</script>
+
+<style scoped>
+
+.inventory {
+  background-color: lightgray;
+  display: flex;
+  border: 1px black solid;
```
Comment (MaxenceMottard) [inline src/reusables/InventoryBarWidget.vue]: needs to use `position: fixed` css property and place the element at the bottom of the screen and centered horizontally

**Review by MaxenceMottard — CHANGES_REQUESTED** [review]

**Comment by MaxenceMottard** [comment]
When your bar is in the fixed position, you can call your component in the Party.vue file before the component with the ID `#debug-button`.

**Comment by MaxenceMottard** [comment]
Please fix bar max-width

Screenshot

**Review by MaxenceMottard — APPROVED** [review]

**Pull request merged**